src/plugins/linked.js

```diff
@@ -16,16 +16,22 @@ type PJSON = {
   }
 }
 
+function touch (f: string) {
+  fs.utimesSync(f, new Date(), new Date())
+}
+
 export default class LinkedPlugins implements IPluginManager {
   constructor (out: Output) {
     this.out = out
     this.config = this.out.config
     try {
       this._data = fs.readJSONSync(this.file)
+      this._data.updated_at = new Date(this._data.updated_at)
```

Comment:
@dickeyxxx could you change this line
```
new Date(this._data.updated_at)
```
to
```
new Date(this._data.updated_at || null)
```
because otherwise the comparison with `fstat` always return false due to `Invalid Date`
```
> new Date(null)
1970-01-01T00:00:00.000Z
> new Date(undefined)
Invalid Date
```